translator/internaldata/traces_to_oc.go

```diff
@@ -386,10 +386,6 @@ func spanIDToOC(sid pdata.SpanID) []byte {
 }
 
 func statusToOC(status pdata.SpanStatus) (*octrace.Status, *octrace.AttributeValue) {
-	if status.IsNil() {
-		return nil, nil
-	}
-
```

Comment:
Should we return (nil, nil) if `status.Code()==case pdata.StatusCodeUnset && status.message==""`?

Reply:
Not sure, was thinking about that as well. Is that not a different change? That behavior was not there before. Do you suggest to do this?

Reply:
Because we now don't have a pointer to Status this case will happen always for all successful statuses, so it is a bit more important now. But, probably doesn't matter much, it only affects OpenCensus format. Let's leave as is and we can change in the future if needed.